apps/CEventService.API/DAO/IRepository.cs

```diff
@@ -0,0 +1,9 @@
+namespace CEventService.API.DAO
+{
+    public interface IRepository<T>
```

Comment:
The abstraction is interesting but the names of the methods makes it impossible to reuse them in other repository implementations, if we instantiate a UsersMongoRepository implementing this interface having a get all events method would not make sense, use more generic names like getAll or getByID and so on, it would also be good to add documentation to specify how it should be implemented and used, something short, for example what should happen with the skip and date of the GetAll, you can overunderstand but it is never too much to have some documentation to be all in the same page.

Reply:
Let me understand well, you are asking to erase the word asyn aftert each method name?
Also the skip and take parameters are well documented in its official docs. Do you want me to copy them as a comment in the file? @CatCoder36

Reply:
About the name the fact that it says “getAllEvents” makes the IRepository interface only makes sense to implement it in an EvenRespository, as it is an interface that in theory should implement it in other repositories a more generic name is necessary and about the documentation of the parameters that is documentation for the Api consumer, but the documentation in code is necessary for the devs. @victor-fundacion

Reply:
Thx for aclaring my doubt, I'll modified the interface to handle generic names and add a comment on explaining what skip and take are for. Is that ok @CatCoder36 ??